app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<link rel="stylesheet" href="../assets/stylesheets/normalize.css">
```

Comment:
You shouldn't need to add this file here -- we haven't gone over how to have different stylesheets, but in the future, it will be included